org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -382,3 +382,39 @@ Resources:
           - SecurityHubFindingsQueue
           - Arn
         Id: Target0
+
+  # This rule suppresses findings with the given generator IDs only for the bootstrap account and service accounts
+  SuppressFindingsForPublicBucketsRule:
+    Type: AWS::Events::Rule
+    Properties:
+      Description: SecHubSuppress findings (identified by generatorId) for the bootstrap and essentials buckets in all accounts
```

Comment:
seems like this is doing more than just boostrap and essentials?

Reply:
Put a more generic description.